website/docs/guides/docs/docs-create-doc.mdx

```diff
@@ -137,7 +137,7 @@ slug: /bonjour
 Lorem ipsum
 ```
 
-`slug` will be appended to the doc plugin's `routeBasePath`, which is `/docs` by default. See [Docs-only mode](#docs-only-mode) for how to remove the `/docs` part from the URL.
+`slug` will be appended to the doc plugin's `routeBasePath`, which is `/docs` by default. See [Docs-only mode](./docs-introduction#docs-only-mode) for how to remove the `/docs` part from the URL.
```

Comment:
```suggestion
`slug` will be appended to the doc plugin's `routeBasePath`, which is `/docs` by default. See [Docs-only mode](docs-introduction.md#docs-only-mode) for how to remove the `/docs` part from the URL.
```

Reply:
Thanks

We use relative file path links as recommended in our docs to avoid links being trailingSlash sensitive